gm4_apple_trees/data/gm4_fruiting_trees/functions/main.mcfunction

```diff
@@ -1,4 +1,7 @@
 # age fruiting leaves
 execute as @e[type=armor_stand,tag=gm4_fruiting_leaf] run function gm4_fruiting_trees:tree/leaf/fruiting/age
 
+# register trades from expansions on wandering traders
+execute as @e[type=wandering_trader,tag=!gm4_fruiting_trader] at @s run function gm4_fruiting_trees:wandering_trader/add_trade
```

Comment:
this may want to use `tag=!` instead of `tag=!gm4_fruiting_trader` so it doesn't potentially mess up future modules that want to provide fully custom wandering traders.

Reply:
I thought about it, but that would break this entirely in case another module does something with tags on wandering traders for internal reasons.
Such modules that don't want their trades messed with could simply add `gm4_fruiting_trader` in advance.
We could consider renaming this tag to `gm4_ignore`.